src/components/Dataset.tsx

```diff
@@ -114,6 +142,18 @@ export const Dataset = (): JSX.Element => {
 		}
 	];
 
+
+	if (folderPath != null) {
+		for (const folderBread of folderPath) {
+			paths.push({
+				"name": folderBread["folder_name"],
+				"url": `/datasets/${datasetId}?folder=${folderBread["folder_id"]}`
+			})
+		}
+	} else {
+		paths.slice(0, 1)
```

Comment:
could you remind me when this will be triggered? Folderpath seems to have a default of []?

Reply:
If you traverse to a nested folder and then click on root dataset the path in the breadcrumb should still be there without this.